packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.controller.js

```diff
@@ -147,4 +153,13 @@ export default class SoftphoneController {
         ),
       );
   }
+
+  deleteLogo() {
+    this.softphoneService.putSoftphoneLogo(
+      this.billingAccount,
+      this.serviceName,
+      '',
+      '',
+    );
```

Comment:
You can remove the useless parameters IMO
```suggestion
  deleteLogo() {
    this.softphoneService.putSoftphoneLogo(
      this.billingAccount,
      this.serviceName,
    );
```

Reply:
Nope it won't work with api

Reply:
Ok but in that case your function should be define like that : 
```
function putSoftphoneLogo(billingAcount, serviceName, param3 = '', param4 = '')
```
To avoid passing empty arguments each time

Reply:
More elegant indeed :)